packages/react-native-win32/package.json

```diff
@@ -13,9 +13,9 @@
     "lint:fix": "just-scripts lint:fix",
     "lint": "just-scripts lint",
     "api": "just-scripts api",
-    "run-win32-dev-web": "rex-win32 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev --useDevMain --useWebDebugger",
-    "run-win32-devmain": "rex-win32 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev --useDevMain",
-    "run-win32": "rex-win32 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev",
+    "run-win32-dev-web": "npx @office-iss/rex-win32@0.62.7-tenantreactnativewin-13222 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev --useDevMain --useWebDebugger",
+    "run-win32-devmain": "npx @office-iss/rex-win32@0.62.7-tenantreactnativewin-13222 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev --useDevMain",
+    "run-win32": "npx @office-iss/rex-win32@0.62.7-tenantreactnativewin-13222 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev",
```

Comment:
these aren't needed for RNW apps, is this something that can live on the office side instead?

Reply:
applies to the whole react-native-win32 folder in reality, but yeah :)